components/apimgt/org.wso2.carbon.apimgt.rest.api.admin/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/impl/WorkflowsApiServiceImpl.java

```diff
@@ -0,0 +1,94 @@
+package org.wso2.carbon.apimgt.rest.api.admin.impl;
```

Comment:
Let's add the license header

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/7064/commits/27b620bccf7276e607a5673c66dd07a4706a620f